etc/ansible/roles/network-runner/providers/dellemc.os9.os9/conf_trunk_port.yaml

```diff
@@ -22,13 +23,15 @@
     lines:
       - "interface vlan {{ _vlan_id }}"
       - "untagged {{ port_name }}"
+    save: true
   connection: network_cli
 
 - name: "dellemc.os9: add trunk vlan(s)"
   dellemc.os9.os9_config:
     lines:
       - "interface vlan {{ t_vlan }}"
       - "tagged {{ port_name }}"
+    save: true
```

Comment:
I am worried about the performance impact here since this call will be in a loop. Imagine if we had to add 10 vlans to the trunk, then it will also call save 10 times.

Maybe we don't save config in this step and move the call to set native vlan after this step and rely on saving in that step? Actually, what if we add some noop config task at the end of the playbook with the sole purpose of saving the config? That way we only save the config when it's completely applied every thing in the playbook?

Reply:
You're right I didn't notice the loop. Should we just have 1 save task at the end of each playbook then to keep it consistent?

Reply:
> Should we just have 1 save task at the end of each playbook then to keep it consistent?

Yes, that's what I think. Essentially we save when the playbook is complete instead of each individual task.

Reply:
Done